packages/infrastructure/webpack/src/get-react-config.js

```diff
@@ -1,13 +1,16 @@
 const getNodeConfig = require("./get-node-config");
 const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const path = require("path");
+
 
 module.exports =
-  ({ miniCssExtractPluginLoader = MiniCssExtractPlugin.loader } = {}) =>
+  ({ miniCssExtractPluginLoader = MiniCssExtractPlugin.loader, dependencies = { resolvePath: path.resolve, scriptDirectory: __dirname } } = {}) =>
   ({ entrypointFilePath, outputDirectory }) => {
     const nodeConfig = getNodeConfig({
       entrypointFilePath,
       outputDirectory,
     });
+    const sassCommonVars = dependencies.resolvePath(dependencies.scriptDirectory, "..", "..", "..", "core", "src", "renderer", "components/vars.scss");
```

Comment:
Here we're importing stuff that belongs to some other package. Does this even work when the package is built and used in some other repository?

Reply:
I'm not sure. Probably not. I'll remove this reference altogether. It's bring you own SASS variables config!

Reply:
removed it